Add a prefix to tags cache file names (#751)
DataDog/datadog-serverless-functions: aws/logs_monitoring/README.md

### JSON-formatted logs are not appearing in Datadog
If your logs contain an attribute that Datadog parses as a timestamp, you need to make sure that the timestamp is both current and in the correct format. See [Log Date Remapper][24] to learn about which attributes are parsed as timestamps and how to make sure that the timestamp is valid.

### Issue creating S3 triggers

> In the case of encounteirng the below error when creating S3 triggers related to using prefix filtering. We recommend considering following a fanout architecture proposed by AWS [in this article](https://aws.amazon.com/blogs/compute/fanout-s3-event-notifications-to-multiple-endpoints/)

Reviewer: ```suggestion
In case you encounter the following error when creating S3 triggers, we recommend considering following a fanout architecture proposed by AWS [in this article](https://aws.amazon.com/blogs/compute/fanout-s3-event-notifications-to-multiple-endpoints/)
```

Reviewer: also it could be useful to mention [here](https://github.com/DataDog/datadog-serverless-functions/pull/751/files#diff-6a31213916423d2eee54f995d4a33652ee5aa7b54c88285ad4ff8d239d9399acR123) that 

Since version xx of the forwarder, this bucket can be shared between several AWS lambda definitions of the forwarder since the cache files will be store under a different prefix in the S3 bucket for different lambdas.

Or sth like that ...
